Add Odido icon

xxsuperdreamxx opened the pull request:
<!--
Before opening your pull request, have a quick look at our contribution guidelines:
https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md

Consider adding a preview image of your submission using:
https://simpleicons.org/preview
-->
![odido](https://github.com/user-attachments/assets/a350425f-e8d6-4554-bd99-b45a71f692e6)


**Issue:** closes #12442

**Popularity metric:** Source https://www.similarweb.com/website/odido.nl/
| Global Rank | Country Rank |
|-------------|--------------|
| 8,220       | NL: 117      |

<!--
Regardless of whether or not the linked issue (if there is one) has a metric, please include the metric here for PR reviewers to validate. See our contributing guidelines at https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#assessing-popularity for more details on how we assess a brand's popularity.
-->

### Checklist

- [x] I have reviewed the [forbidden brands](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#forbidden-brands) list and confirm the brand I am submitting a PR for is not one of them, nor is it a subsidiary of one of those brands
- [x] I have reviewed the brand's terms of service, and am confident we can add this icon
- [x] I updated the JSON data in `data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

xxsuperdreamxx committed:
Add Odido icon (#12442)

xxsuperdreamxx committed:
Merge branch 'develop' into odido

PeterShaggyNoble commented on data/simple-icons.json:
```diff
@@ -11382,6 +11382,20 @@
+	{
+		"title": "Odido",
+		"hex": "000000",
+		"source": "https://www.odido.nl/Assets/Lfwd/Static/odido-logo-black.svg",
```
For better future proofing, we use the URL of the page the source file was found but, in this case, the logo page of the brand guidelines works better.
```suggestion
		"source": "https://merk.odido.nl/en/logo",
```

PeterShaggyNoble commented on data/simple-icons.json:
```diff
@@ -11382,6 +11382,20 @@
+		"title": "Odido",
+		"hex": "000000",
+		"source": "https://www.odido.nl/Assets/Lfwd/Static/odido-logo-black.svg",
+		"guidelines": "https://merk.odido.nl/en/logo",
```
We should link to the full brand guidelines here.
```suggestion
		"guidelines": "https://merk.odido.nl/en",
```

PeterShaggyNoble commented on data/simple-icons.json:
```diff
@@ -11382,6 +11382,20 @@
+		"guidelines": "https://merk.odido.nl/en/logo",
+		"aliases": {
+			"aka": [
+				"ODIDO"
```
This alternative casing alias is unnecessary. And I also don't see them referring to themselves this way anywhere.

PeterShaggyNoble commented on data/simple-icons.json:
```diff
@@ -11382,6 +11382,20 @@
+				"ODIDO"
+			],
+			"old": [
+				"T-Mobile"
```
We only use the `old` alias when updating existing icons in our library.

xxsuperdreamxx replied:
Got it, since people still Refer to Odido as T-Mobile that's why

PeterShaggyNoble reviewed (changes requested):
I can't get you version to line up with any of the ones they make available to download through their brand centre, @xxsuperdreamxx. We should use one of the EPSes available in the `Millen_URL_SolidLogo_SmallUseALL` zip found on [this page](https://dam.odido.nl/web/52967ad9a20e29eb/b2c-logos---downloads-brand-portal/?viewType=grid). See also notes below on metadata.

xxsuperdreamxx committed:
add Odido Icon with updated source and guidelines

- Based icon on Millen_URL_SolidLogo_SmallUse_Black.eps
- Removed aliases from icon data
- Updated source and brand guidelines URLs

adamrusted commented:
Just spotted on the source page:

> Minimum size of the small use logo is 60px or 16mm.

We cannot guarantee the minimum size will be used, and in fact we store images at 24px - so we are going against their terms by storing the icon in that style.

PeterShaggyNoble commented:
@adamrusted, have we not worked around that in the past through our inclusion of brand guidelines and requirement that users read & adhere to them. If they don't then that's on them not us. And I wouldn't consider Shields.io to be exempt from that requirement.
